wiki/Gameplay/Accuracy/en.md

```diff
@@ -23,23 +23,40 @@ Reference for one hit circle:
 
 ### ![](/wiki/shared/mode/taiko.png) osu!taiko
 
-![Accuracy = 0.5(number of GOOD + number of GREAT) / (number of BAD + number of GOOD + number of GREAT)](img/accuracy_taiko.png "Accuracy formula for osu!taiko")
+![Accuracy = (number of GREATs + 0.5 \* number of GOODs) / (number of GREATs + number of GOODs + number of misses)](img/accuracy_taiko_updated.png "Accuracy formula for osu!taiko")
 
 In osu!taiko, accuracy is calculated by taking the sum of the note accuracy (how close you were to hitting the note on time) divided by the number of total notes scored thus far. The note accuracies are labelled as a GREAT (良) (counts as 100%), GOOD (可) (counts as 50%) (half), and MISS/BAD (不可) (counts as 0%, which also breaks the combo). Drum rolls and spinners do not influence accuracy.
 
 ### ![](/wiki/shared/mode/catch.png) osu!catch
 
-![Accuracy = (number of droplets + number of drops + number of fruits) / (number of missed droplets + number of missed drops + number of missed fruits + number of droplets + number of drops + number of fruits)](img/accuracy_catch.png "Accuracy formula for osu!catch")
+![Accuracy = (number of caught fruits + number of caught drops + number of caught droplets) / (number of all fruits + number of all drops + number of all droplets)](img/accuracy_catch_updated.png "Accuracy formula for osu!catch")
 
 In osu!catch, accuracy is calculated by taking the total number of non-spinner hit objects collected, divided by the total number of non-spinner objects. All hit objects have the same value; except for bananas, as they are part of the spinner objects.
 
-*Note for API users: To calculate the accuracy in osu!catch, the number of droplets is under `count50` and the number of missed droplets is under `countkatu`.*
+*Notes for API users:*
+
+- The number of caught drops is returned as `count100`.
+- The number of caught droplets is returned as `count50`.
+- The number of missed fruits *and* drops cumulatively is returned as `countMiss`.
+- The number of missed droplets is returned as `countKatu`.
+- `countGeki` should not be used to calculate the accuracy at all. It is the count of caught combo-ending fruits.
```

Comment:
Which API? osu! API v1 or v2 or both?

Reply:
Doesn't matter in this context as things stand. `countGeki` and `countKatu` are present in both versions. However I'm pretty sure api v2 will be eventually migrating to something saner at some point.

Reply:
Alright add api as hyperlink to this article https://osu.ppy.sh/wiki/en/osu!api

Reply:
```suggestion
*Notes for [API](/wiki/osu!api) users:*

- The number of caught drops is returned as `count100`.
- The number of caught droplets is returned as `count50`.
- The number of missed fruits *and* drops cumulatively is returned as `countMiss`.
- The number of missed droplets is returned as `countKatu`.
- `countGeki` should not be used to calculate the accuracy at all. It is the count of caught combo-ending fruits.
```